furms-web-ui/src/main/java/io/imunity/furms/ui/components/FurmsUserComboBox.java

```diff
@@ -21,15 +21,14 @@
 public class FurmsUserComboBox extends CustomField<FurmsViewUserModel> {
 	private final CustomValueCheckBox<FurmsViewUserModel>  comboBox = new CustomValueCheckBox<>();
 
-	public FurmsUserComboBox(List<FurmsViewUserModel> userModels) {
+	public FurmsUserComboBox(List<FurmsViewUserModel> userModels, boolean allowCustomValue) {
```

Comment:
it would be good to add static c-tor builders, it is puzzling to see in the code 
`new FurmsUserComoBox(list, true)`
It would increase code readability to have something like:
`FurmsUserComoBox.withUserInputEnabled(list)`
`FurmsUserComoBox.withUserInputDisabled(list)`